pyked/chemked.py

```diff
@@ -255,7 +257,7 @@ def get_dataframe(self, output_columns=None):
         if output_columns is None or len(output_columns) == 0:
             col_labels = valid_labels
             comp_index = col_labels.index('composition')
-            col_labels[comp_index:comp_index + 1] = species_list
+            col_labels[comp_index:comp_index + 1] = species_list + ['Kind']
```

Comment:
Can you call this `Composition:Kind` to disambiguate from `Apparatus:Kind`?

Reply:
You've got it, I also squashed the commits so it's cleaner

Reply:
@arghdos that will make @bryanwweber very happy :)